src/book/04-general/01-git.mdx

```diff
@@ -0,0 +1,59 @@
+
+Github has documentation about PRs [here.](#https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
+
+Title your PR with a description of the changes that it will make. For this example we could use `Add stand-up script`. If you need to give more detail about what the code does, put that as a comment.
+
+To be approved, you should add reviewers to the pull request. If you have a project mentor, they should be a reviewer, as well as someone else who knows about that part of the codebase. Once the reviewers have approved the changes, the changes can be merged into the master branch.
+
+### Code Review
+
+The reviewers will give constructive feedback about your code, by making comments on code snippets from your changes. You can respond to these on Github. They can also directly suggest changes to specific sections. Additional commits you make to the branch will update the PR and will reset their review of changed sections.
+
+Once the reviewers are happy with it, and all of the comments are resolved, you can `Squash and Merge`. Squashing removes detailed commit descriptions, so that `master` is not messy with every commit ever. Once the code has been squashed and merged, it is in the codebase and you have succeeded in contributing code to NUbots.
+
+## Commit Etiquette
+
+Commits should make minor changes, and the changes in each commit should have a common theme. A general mantra for contributing code is `commit often, push once`, but this isn't strict at all. Committing often means that the code is built up incrementally. Pushing once means that you don't spam all the other contributors with notifications.
+
+## Formatting and buildkite
+
+Before making a pull request, it is important that your code adheres to the formatting rules. For branches in the NUbots repo, running `./b format` before your last commit should fix formatting issues. Code in the NUbook repo must be formatted with `yarn format` before it can be committed.
+
+If your code is in a different repository and it uses C++ or protobuf files, use the `.clang-format` file in the NUbots repo to format them.
+
+If you get a message that your build failed, try running the relevant format commands and then committing the changes which they made. You won't be able to merge the PR until the build checks pass.
+
+## Work in Progress PRs
+
+If you have code which isn't done yet, but which you would like feedback on, you can make a draft PR to get your code reviewed. Prepend `[WIP]` to the PR title when making draft pull requests. Draft PRs cannot be merged, so there is no need to worry about merging in bad code. Github has documentation about draft PRs [here.](#https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests#draft-pull-requests)
```

Comment:
there is a button you can press for this (drop down on 'create pull request' then change to draft pull request)

Reply:
I was steering clear of explicitly saying which button and where the button is, but we can add if you reckon

Reply:
Ye fair enough and you link to the GitHub page about it anyway